zuul.d/jobs.yaml

```diff
@@ -22,10 +22,26 @@
     nodeset: centos-8-1vcpu
 
 - job:
-    name: ansible-tox-molecule
-    parent: tox-molecule
+    name: ansible-tox-docs
+    parent: tox-docs
+    nodeset: centos-8-1vcpu
+
+- job: &_stub_molecule
+    name: _stub_molecule
+    # Stub job used to define only properties specific to molecule jobs,
+    # expected to decorate other job definitions via anchors. To avoid overrides
+    # it does not have: nodeset, abstract, description
     pre-run: playbooks/ansible-tox-molecule/pre.yaml
     post-run: playbooks/ansible-tox-molecule/post.yaml
+    timeout: 5400  # 1.5h as they are expected to be slower by nature
+
+- job:
+    <<: *_stub_molecule
```

Comment:
I think molecule-tox-base would be better, and fix into current naming scheme.

Reply:
The reson why I used `_stub_molecule` is to follow python private var pattern (underline prefix) because I am unable to use abstract on it.
Also I wanted to avoid poluting list of job in job-tree in zuul. Thus is why opeted for that name, making it more obvious that they should never try to use that partial/incomplet job.

Whatever name we pick, it should better start with a prefix that indicates that is not a normal job. I wonder if we can use dot, like on linux. Bad part about dot is that is listed before "a".

If we are to look read, https://stackoverflow.com/questions/8086375/what-character-to-use-to-put-an-item-at-the-end-of-an-alphabetic-list we are better of using `z_`, I think.